skbio/stats/distance/_base.py

```diff
@@ -684,12 +707,42 @@ def _subset_to_dataframe(
         # included here so that np.hstack works in the event that either i_ids
         # or j_ids is empty.
         values = [np.array([])]
-        for i_idx in i_indices:
-            i.extend([self.ids[i_idx]] * j_length)
-            j.extend(j_labels)
+        if self._flags["CONDENSED"]:
```

Comment:
Is this code under `PairwiseMatrix` or `SymmetricMatrix`? I assume the later, because the former doesn't have a condensed form. Let me know if I am wrong.